ios-target/View/Home/TargetFormViewController.swift

```diff
@@ -10,7 +10,65 @@ import UIKit
 
 class TargetFormViewController: ModalViewController {
 
+  @IBOutlet weak var areaTextField: UITextFieldPadding!
+  @IBOutlet weak var titleTextField: UITextFieldPadding!
+  @IBOutlet weak var topicTextField: UITextFieldPadding!
+  @IBOutlet weak var createTargetButton: UIButton!
+  
+  var viewModel = TargetFormViewModel()
+  
+  weak var delegate: TargetActionsDelegate?
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    viewModel.delegate = self
+    setCreateTargetButton(enabled: false)
+  }
+  
+  @IBAction func formEditingChange(_ sender: UITextField) {
+    let newValue = sender.text ?? ""
+    switch sender {
+    case areaTextField:
+      viewModel.targetArea = Double(newValue) ?? 0
+    case titleTextField:
+      viewModel.targetTitle = newValue
+    case topicTextField:
+      viewModel.targetTopic = Int(newValue) ?? 0
```

Comment:
why are you expecting an int in the topic text field ?

Reply:
shouldn't it be a picker?

Reply:
yep, missing topics though, comming on next pr 😊

Reply:
ah makes sense! 💪